### PR TITLE
Remove outdated tests in mocha - Closes #4882

### LINKED ISSUE #4882: Remove unused mocha tests
### Description

There are a lot of outdated skipped tests in mocha, and that should be cleaned.
After removing the outdated test, unused utils files should be removed.

### Motivation

Clean up of the tests

### Acceptance Criteria

In the mocha test folder, only used and not outdated should exist.
These are the lists that is removed partially or completely
```
framework/test/mocha/functional/http/get/dapps.js
framework/test/mocha/functional/http/get/transactions.js
framework/test/mocha/functional/http/post/0.transfer.js
framework/test/mocha/functional/http/post/5.dapps.js
framework/test/mocha/functional/http/post/6.dapps.in_transfer.js
framework/test/mocha/functional/http/post/7.dapps.out_transfer.js
framework/test/mocha/integration/blocks/chain/delete_last_block_accounts.js
framework/test/mocha/integration/blocks/process/on_receive_block.js
framework/test/mocha/integration/blocks/process/process.js
framework/test/mocha/integration/blocks/verify/verify.js
framework/test/mocha/integration/loader/validate_own_chain/first_round.js
framework/test/mocha/integration/loader/validate_own_chain/full_two_rounds.js
framework/test/mocha/integration/loader/validate_own_chain/more_than_two_rounds.js
framework/test/mocha/integration/loader/validate_own_chain/valid_chain_full_round.js
framework/test/mocha/integration/loader/validate_own_chain/valid_chain_partial_round.js
framework/test/mocha/network/index.js
framework/test/mocha/network/network.js
framework/test/mocha/network/scenarios/propagation/blocks.js
framework/test/mocha/network/scenarios/propagation/multisignatures.js
framework/test/mocha/network/scenarios/propagation/transactions.js
framework/test/mocha/network/setup/config.js
framework/test/mocha/network/setup/index.js
framework/test/mocha/network/setup/shell.js
framework/test/mocha/network/utils/http.js
framework/test/mocha/network/utils/index.js
framework/test/mocha/network/utils/transactions.js
framework/test/mocha/network/utils/ws.js
framework/test/mocha/unit/application/node/init_steps/bootstrap_cache.js
framework/test/mocha/unit/application/node/init_steps/bootstrap_storage.js
framework/test/mocha/unit/application/node/init_steps/create_bus.js
framework/test/mocha/unit/application/node/init_steps/init_logic_structs.js
framework/test/mocha/unit/application/node/submodules/app.js
framework/test/mocha/unit/application/node/submodules/loader.skeleton.js
framework/test/mocha/unit/application/node/utils/config.js
```

### PR DESCRIPTION
**Pull request by shuse2**
### What was the problem?

This PR resolves #4882 

### How was it solved?

Removed outdated tests and skipped tests

### CODE REVIEW

File: framework/test/mocha/unit/application/node/utils/config.js [hunk 1]
```diff
@@ -1,142 +0,0 @@
...
-
-/* eslint-disable */
-
-// This file is not used any more will be replaced completely with https://github.com/LiskHQ/lisk/issues/2976
-// const config = require('../../../../../../src/modules/chain/helpers/config');
-const { ZSchema } = require('../../../../../../src/application/validator');
-
-// This file is not used any more
-// eslint-disable-next-line
-describe.skip('helpers/config', () => {
```
Comment (ManuGowda) [inline framework/test/mocha/unit/application/node/utils/config.js]: do we have some tests covering this configuration helper?

Comment (shuse2) [reply framework/test/mocha/unit/application/node/utils/config.js]: we don't have this script anymore =)

Comment (nazarhussain) [reply framework/test/mocha/unit/application/node/utils/config.js]: Individual cases are covered in separate unit tests for keywords.

File: framework/test/mocha/integration/blocks/process/on_receive_block.js [hunk 2]
```diff
@@ -17,7 +17,6 @@
 const expect = require('chai').expect;
```
[the remark this answers is not in the document]
Comment (shuse2) [reply framework/test/mocha/integration/blocks/process/on_receive_block.js]: `fork1` and `fork5` doesn't exist, and the fork is handled in the BFT specs and unit tests

**Comment by shuse2** [comment]
For the network test, we can improve the description in https://github.com/LiskHQ/lisk-sdk/issues/3389, but this needs further brainstorming, what we want to test and how we want to test them

**Pull request merged**